test_eos-phone-home.py

```diff
@@ -0,0 +1,159 @@
+import imp
+import os
+
+import pytest
+
+HERE = os.path.dirname(__file__)
+eos_phone_home_path = os.path.join(HERE, 'eos-phone-home')
+eos_phone_home = imp.load_source("m", eos_phone_home_path)
+
+
+@pytest.fixture
+def phone_home(tmpdir):
+    return eos_phone_home.PhoneHome(True, False, root=tmpdir.strpath)
+
+
+def test_instantiate(tmpdir, phone_home):
+    assert tmpdir.strpath == phone_home._root
+
+
+def test_get_product_info_arm(tmpdir, phone_home):
+    compatible = tmpdir.join('proc', 'device-tree', 'compatible')
+    compatible.write_binary(b'vendor1,model1\0vendor2,model2\0',
+                            ensure=True)
+
+    ssn = tmpdir.join('sys', 'class', 'endless_mfgdata', 'entries', 'SSN')
+    ssn.write('  serial  ', ensure=True)
+
+    pi = phone_home._get_product_info()
+    assert pi == eos_phone_home.ProductInfo('vendor1', 'model1', 'serial')
```

Comment:
So this goes back a bit to `phone_home` vs `eos_phone_home` var naming but it's unclear what is being tested here. ditto on the other tests on this.